app/views/restaurant/show.html.erb

```diff
@@ -1,10 +1,15 @@
 <h2><%= @restaurant.name %></h2>
 <p><%= @restaurant.description %></p>
 
-<% @restaurant.products.each do |product| %>
-  <div id="product-<%= product.id %>">
-     <h4><%= product.name %></h4>
-     <p><%= product.description %></p>
-     <p><%= product.price %></p>
-  </div>
+<% @restaurant.product_categories.each do |product_category| %>
+<div id="product_category-<%= product_category.id %>">
+  <h3><%= product_category.name %></h3>
+  <% product_category.products.each do |product| %>
```

Comment:
To use the Swedish expression “nja...” this fetches ALL products in this category. We only want to see the products that be lotto this specific restaurant. You must revisit this

Reply:
A product_category only belongs to a specific restaurant, so only the products for the restaurant in question are showing.